stripe/src/main/java/com/stripe/android/model/Stripe3ds2Fingerprint.java

```diff
@@ -44,15 +52,70 @@ static Stripe3ds2Fingerprint create(@NonNull JSONObject json) throws JSONExcepti
         final DirectoryServer directoryServer =
                 DirectoryServer.lookup(json.getString(FIELD_DIRECTORY_SERVER_NAME));
         final String serverTransactionId = json.getString(FIELD_SERVER_TRANSACTION_ID);
-        return new Stripe3ds2Fingerprint(source, directoryServer, serverTransactionId);
+        final DirectoryServerEncryption directoryServerEncryption =
+                DirectoryServerEncryption.create(
+                        json.optJSONObject(FIELD_DIRECTORY_SERVER_ENCRYPTION));
```

Comment:
let's use `get` instead of `opt` - this field should always be populated and is required for 3DS2